external-providers/golang-external-provider/pkg/golang/dependency.go

```diff
@@ -16,6 +16,17 @@ const (
 	golangDownloadableDepSourceLabel = "downloadable"
 )
 
+// createDepSourceLabel creates a dependency source label with the specified value.
+func createDepSourceLabel(label, value string) string {
```

Comment:
These two functions do the exact same thing. We should use the  exposed function from labels package instead.